tools/generate_hotkeys.py

```diff
@@ -127,4 +131,11 @@ def various_key_combination(key_combinations_list: List[str]) -> str:
 
 
 if __name__ == "__main__":
-    main()
+    parser = argparse.ArgumentParser(description="Lint and generate hotkeys")
```

Comment:
description=" Generate or update hotkeys.md file based on changes in config/keys.py by default. To lint hotkeys add '--check-only' option."
How about this description?

Reply:
The first sentence is all that's necessary - I think this is only shown in the help text, and in that case the option will be listed below. So if the option is well-described, we don't need to include it here.

The 'by default' could be in parentheses, perhaps? As I mentioned elsewhere, it would be cleaner to use string substitution for the file names which are used throughout, and likely refer to the variables in docstrings instead.
(alternatively, some of those functions could take in file parameters, but let's leave that change for this PR)